dropwizard-parent-pom/pom.xml

```diff
@@ -23,13 +23,20 @@
     <dropwizard.version>1.0.5</dropwizard.version>
 
     <!--
+      This version needs to stay EXACTLY the same as the one that the Dropwizard BOM
+      specifies.  Unfortunately when you import a BOM you don't get its properties, only
+      its dependencies.
+    -->
+    <jackson.version>2.7.8</jackson.version>
```

Comment:
Dropwizard 1.1 includes the parameter names jar so we will be able to remove this again when we upgrade